### PR TITLE
Added Wallet Selector in sidebar with all wallet docs

### PR DESCRIPTION
**Pull request by amirsaran3**
Added "Wallet Selector" category to "Accounts & Transactions" sub-section on "Tools" page. Category has docs for each currently available wallet.

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -62,9 +62,9 @@ module.exports = {
           anonymizeIP: true,
         },
         blog: {},
-        theme: {
-          customCss: "/src/css/customTheme.css",
-        },
+        // theme: {
+        //   customCss: "/src/css/customTheme.css",
+        // },
```
Comment (bucanero) [inline website/docusaurus.config.js:67]: why are you disabling the CSS theme?

**Review by bucanero — CHANGES_REQUESTED** [review]
please restore the CSS theme

File: website/docusaurus.config.js (hunk 1 above)
Comment (bucanero) [inline website/docusaurus.config.js:67]:
```suggestion
         theme: {
           customCss: "/src/css/customTheme.css",
         },
```

**Comment by gagdiez** [comment]
@amirsaran3 this looks like low level (API reference) documentation. I would strongly recommend to put it on its own website/github pages.

In our docs we will only want a high level explanation on how to use the wallet selector.

File: website/docusaurus.config.js (hunk 1 above)
Comment (amirsaran3) [reply website/docusaurus.config.js:67]: Hey. Sorry for not responding. I had problems running the development server while this theme was here. So I removed it just to be able to work. I pushed new changes that does not remove it.

**Pull request closed without merging**